src/models/user.ts

```diff
@@ -66,13 +67,9 @@ User.init(
       allowNull: true,
       defaultValue: ""
     },
-    createdAt: {
```

Comment:
Why did you remove the time stamp for this model?

Reply:
there I removed the timestamp to use the timestamp from sequelize, those codes were doing the same thing as a timestamp: true which was in that sequelize options object below